docs/hardware/devices/heltec/index.mdx

```diff
@@ -82,15 +87,19 @@ This device may have issues charging a connected battery if utilizing a USB-C to
 
 Image Source: [Heltec](<https://resource.heltec.cn/download/WiFi_LoRa32_V3/HTIT-WB32LA(F)_V3.png>)
 
-## Resources
+### Resources
 
 - Firmware file: `firmware-heltec-v3-X.X.X.xxxxxxx.bin`
-- Purchase link: [Heltec](https://heltec.org/project/wifi-lora-32-v3/)
-- Purchase link: [AliExpress](https://www.aliexpress.us/item/3256805256690400.html)
+- Purchase links
```

Comment:
This purchase link header was added for V3 but is missing from Heltecs WSL, Tracker, & Paper.

Also missing from all T-Beams other than Supreme

Reply:
I can follow up with updates here, or remove these updates from the current. Any preference? I do think there are other updates to standardize beyond what we've done here and are talking about. Wanted to focus on headings mostly in this PR.

Reply:
I like the new format you came up with, just given the PR title I thought maybe you skipped the others accidentally.

Reply:
Thank you. Not an accidental miss, I just didn't want to go too deep in on my first contribution in case it was all shot down anyways.  I plan to follow up with more PRs.